tests/test_emails.py

```diff
@@ -322,6 +332,7 @@ def test_send_email(self):
             msg = msg.format(localmachine=platform.node())
```

Comment:
`test_only_one_project` is defined here twice

Reply:
One instance removed